CIP-0001/README.md

```diff
@@ -242,40 +250,48 @@ This must be subdivided into two sub-sections:
 
 CIPs are classified into distinct categories that help organise (and thus, find) them. Categories are meant to be flexible and evolve as new domains emerge. Authors may leave the category as `?` should they not be sure under which category their proposal falls; editors will eventually assign one or reject the proposal altogether should it relate to an area where the CIP process does not apply.
 
-At present, we consider the following list of initial categories:
-
-Category               | Description
----                    | ---
-Meta                   | Designates meta-CIPs, such as this one, which typically serves another category or group of categories.
-Wallets                | For standardisation across wallets (hardware, full-node or light).
-Tokens                 | About tokens (fungible or non-fungible) and minting policies in general.
-Metadata               | For proposals around metadata (on-chain or off-chain).
-Tools                  | A broad category for ecosystem tools not falling into any other category.
+Submissions can be made to these categories whenever relevant, without following any particular submission guidelines:
 
-Additionally, projects of the ecosystem may explicitly enlist as new categories. The following section describes how projects can engage with the CIP process.
+Category | Description
+---      | ---
+Meta     | Designates meta-CIPs, such as this one, which typically serve another category or group of categories
+Wallets  | For standardisation across wallets (hardware, full-node or light)
+Tokens   | About tokens (fungible or non-fungible) and minting policies in general
+Metadata | For proposals around metadata (on-chain or off-chain)
+Tools    | A broad category for ecosystem features not falling into any other category
 
-Registered categories for explicitly enlisted projects are otherwise listed below.
+Additionally, representatives of ecosystem categories may explicitly _enlist_ their categories (see next section) to suggest a closer relationship with the CIP process.  The following categories are confirmed as enslisted according to CIPs which define that relationship:
```

Comment:
```suggestion
Additionally, representatives of ecosystem categories may explicitly _enlist_ their categories (see next section) to suggest a closer relationship with the CIP process. The following categories are confirmed as enlisted according to CIPs which define that relationship:
```
spelling and extra space

Reply:
N.B. the "extra" space is not a problem here: https://github.com/cardano-foundation/CIPs/pull/893#discussion_r1767656606